setup.py

```diff
@@ -18,10 +18,10 @@
     install_requires=[
         "tqdm",
         "scikit-learn",
-        "scipy>=1.5.4,<1.6",
+        "scipy>=1.10.1,<1.11",
         "transformers",
         "torch",
-        "numpy>=1.19.5,<1.20",
+        "numpy>=1.21.6",
```

Comment:
I don't think we should specify the upper bound of numpy version; `pip` should find and install proper numpy version for users' environments.

Reply:
I noticed that to make `evaluation.py` work, we should use `numpy` < 1.24. 
Fixed in https://github.com/princeton-nlp/SimCSE/pull/249/commits/ffd9aad60a4e0fa32d7e361154ddb737244985a2